pkg/universe.dagger.io/python/image.cue

```diff
@@ -0,0 +1,22 @@
+package python
+
+import (
+	"universe.dagger.io/docker"
+)
+
+#Image: {
+	pythonVersion: =~#"\d+(?:\.\d+)"# & (*"3.10" | string)
```

Comment:
Now here's a valid use for [raw strings](https://docs.dagger.io/1226/coding-style/#avoid-raw-strings). 🙂 

But I wouldn't bother with the regex. How about version `3.11-rc`? And since this is `python.#Image & { pythonVersion: ... }` I'd just name it `version`.

```suggestion
	version: *"3.10" | string
```

 Can you also add a simple test for it?

```cue
python.#Image & {
    version: "3.9"
}
```

Reply:
Ah that's fair. I was trying to be tricky with `alpine: true` adding `"-alpine"` and restricting versions in that way, but KISS is probably better. Additionally it lets users pin a _specific_ version of the Python container image if they so desire by setting `version: "@sha256:<whatever>"`. I'll make that change and add the test.